firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -644,6 +642,21 @@ public void setDefaults(@XmlRes int resourceId) {
         });
   }
 
+  /**
+   * Set the log level for this {@link FirebaseRemoteConfig}'s namespace. Log messages at or above
```

Comment:
I think "instance" is a more accurate differentiation than "namespace". Instances happen to be differentiated by namespace, among other things, but this change adds a per-instance log level. Also, these docs are external-facing, but namespace is an internal detail I wouldn't expect 3p customers to be aware of.